tv_apps/browser/style/components.css

```diff
@@ -0,0 +1,41 @@
+/* checkbox */
+
+.settings-horizontal-list-item {
+  display: flex;
+  align-items: center;
+}
+
+input[type="checkbox"].settings-list-checkbox {
+  display: none;
+}
+
+input[type="checkbox"].settings-list-checkbox + label {
+  display: inline-block;
```

Comment:
the checked icon in checkbox is not at the vertical center in browser setting page. I think may use following css to do theese 
{
display: inline-flex;
align-items: center;
justify-content: center;
}